plugins/xtrace/wiring.go

```diff
@@ -1,14 +1,9 @@
-// Package xtrace provides three plugins:
+// Package xtrace provides Blueprint plugin for instrumenting services and collecting X-Trace traces. It provides APIs to be used by the wiring spec to do the following:
 //
-//  1. A modifier plugin to wrap the service with an XTrace wrapper to generate XTrace compatible traces/logs by correctly propagating baggage across service boundaries. Automatically generates and include an xtrace-server instance in a Blueprint application.
-//  2. A plugin to define an xtrace-based logger for a process. Log events are added as reports to the currently active xtrace task, if one exists. If no such task exists, then no events are logged.
+//  1. to wrap the service with an XTrace wrapper to generate XTrace compatible traces/logs by correctly propagating baggage across service boundaries. Automatically generates and include an xtrace-server instance in a Blueprint application.
+//  2.to define an xtrace-based logger for a process. Log events are added as reports to the currently active xtrace task, if one exists. If no such task exists, then no events are logged.
```

Comment:
space after 2. 

compiles with a formatting error currently

make sure you visually check the gomarkdoc readmes for sanity check because small formatting errors can result in stuff that doens't read properly when compiled